test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/DetectRedundantSuppressionsFromXML.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq.Expressions;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Helpers;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Warnings.WarningSuppression
+{
+	[SkipKeptItemsValidation]
+	[ExpectedNoWarnings]
+	[SetupLinkAttributesFile ("DetectRedundantSuppressionsFromXML.xml")]
+	public class DetectRedundantSuppressionsFromXML
+	{
+		public static void Main ()
+		{
+			DetectRedundantSuppressions.Test ();
+		}
+
+		[ExpectedWarning ("IL2121", "IL2109", ProducedBy = ProducedBy.Trimmer)]
```

Comment:
Could you create an issue for the fact that the IL2121 is produced in the wrong place? (it should point into the XML ideally) - and then link the issue here.

Reply:
Created issue and mentioned it in a comment